### PR TITLE
Adding OnyeEbube's zarr use cases

### PR DESCRIPTION
**Pull request by OnyeEbube**

### CODE REVIEW

**Comment by joshmoore** [comment]
In general, :+1: for all the effort. But one heads up: `zarr_visuals.png` doesn't look to actually be a png. Perhaps you meant `.md`?

**Comment by OnyeEbube** [comment]
> In general, 👍 for all the effort. But one heads up: `zarr_visuals.png` doesn't look to actually be a png. Perhaps you meant `.md`?

I have done that, thanks for the correction.

File: _data/OnyeEbube/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+~~~
+np.savetxt('array_L.txt', array_L, delimiter=" ")
+np.savetxt('array_L.csv', array_L, delimiter=",")
+ValueError: Expected 1D or 2D array, got 3D array instead
+~~~
+<p>A link to the full blog <a href="https://towardsdatascience.com/why-you-should-save-numpy-arrays-with-zarr-dabff4ae6c0c">is here</a></p>
+<h2>Zarr Visualizations</h2>
+
+![image](https://user-images.githubusercontent.com/101593852/197403790-96adf693-0530-4430-bb1d-26374c6d2b3d.png)
```
Comment (sanketverma1704) [inline _data/OnyeEbube/README.md:52]: Can you please describe this visualisation? Or add a line to describe the visualisation?

**Comment by sanketverma1704** [comment]
Hi @OnyeEbube. Thanks for sending the PR.

I've added one correction and left a comment above, please have a look at them.

Here are some additional thoughts:

- Can you please delete the screenshot you've added in `../assets/images/`? The screenshots are supposed to be added under `_data/<YOUR-USERNAME>/Screenshots`.
- I saw you've used HTML tags in your `.md` file. Would you like to convert them to GitHub markdown syntax? You can refer to the documentation [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

Let me know if there are any issues. Thanks!

**Comment by OnyeEbube** [comment]
> 

Thank you for the corrections, i have updated my readme file

**Comment by sanketverma1704** [comment]
Thanks for updating the PR @OnyeEbube. It looks better now. Merging this. Feel free to add more stuff as you go. Thanks!

**Pull request merged**